tracer/src/Datadog.Tracer.Native/debugger_method_rewriter.cpp

```diff
@@ -532,6 +552,18 @@ HRESULT DebuggerMethodRewriter::ApplyMethodProbe(
 
     rewriterWrapper.SetILPosition(beforeLineProbe);
 
+    const auto branchTargets = std::move(GetBranchTargets(rewriterWrapper.GetILRewriter()));
```

Comment:
same here

Reply:
Done in 1bed3f68dfca764bd20a60fea21287b9274e0258.